fe2-android/app/src/prod/java/com/github/dedis/student20_pop/Injection.java

```diff
@@ -47,7 +47,12 @@
 import okhttp3.logging.HttpLoggingInterceptor;
 
 public class Injection {
+<<<<<<< HEAD
+
+  private static String SERVER_URL = "ws://10.0.2.2:8080/organizer/client/";
+=======
   private static String SERVER_URL = "ws://10.0.2.2:9000/organizer/client";
+>>>>>>> master
```

Comment:
It seems you had some issues while merging master on your branch, this is why the fe2 build failed. As for the Base64 changes, after making a grep everything seems to be modified

Reply:
Yes, indeed ! I still have to test with Francois's changes before merging anyway, to make sure everything works as expected. The problem is that, with the json schema verification, there are some incompatibilities with Windows (the OS I use). He's working on making it work with Windows so that I can test with the backend. If it's still not ready on Monday, I will ask someone using MacOs to test my branch.